Common/src/main/java/com/tabnineCommon/lifecycle/PluginInstalledNotifier.java

```diff
@@ -1,8 +1,9 @@
 package com.tabnineCommon.lifecycle;
 
 import com.intellij.util.messages.Topic;
-
+@Deprecated
```

Comment:
marked this as depricated in order to migrate to the new broadcaster which holds the state.

There are multiple listeners for this and didn't want to deal with refactoring them to use the new broadcaster as part of this PR